common/djangoapps/enrollment/views.py

```diff
@@ -729,6 +731,10 @@ def post(self, request):
                     enrollment_attributes=enrollment_attributes
                 )
 
+            cohort_name = request.data.get('cohort')
+            if cohort_name is not None:
```

Comment:
What if `cohort_name` is some other falsey value, like the empty string or 0?  Should we still add the user to that cohort?

Reply:
I guess your except block would catch that case, too, so you can ignore this comment.